src/modules/checkboxFunction.js

```diff
@@ -0,0 +1,10 @@
+const checkboxFunc = (li, index) => {
+  li.classList.remove('incorrect');
+  if (index.completed === false) {
```

Comment:
- [ ] Great job with the index numbers. Kindly make sure The task object's values for completed is updated upon user actions. This is because they only turn true/false when checked/unchecked after a task has been added or deleted

Reply:
Hello dear @tufoinnkuo10  I didn't quite get what you were asking for. Can you open your slack and explain this to me please ?

Reply:
when you mark a task as complete and refresh the local storage the status doesnt change to true. but if a new task is added thats when the status changes